### PR TITLE
PLAT-175-onboard-Alibaba-Provider-2

### PR DESCRIPTION
**Pull request by chkp-dolevh**

### CODE REVIEW

File: website/docs/r/cloudaccount_alibaba.html.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Basic usage:
+
+```hcl
+resource "dome9_cloudaccount_alibaba" "test" {
+  name                   = "NAME"
+  operation_mode         = "OPERATION MODE"
+  subscription_id        = "SUBSCRIPTION ID"
+  tenant_id              = "TENANT ID"
+  client_id              = "CLIENT ID"
+  client_password        = "CLIENT PASSWORD"
```
Comment (itzikYeret) [inline website/docs/r/cloudaccount_alibaba.html.markdown]: this is not correct

File: website/docs/r/cloudaccount_alibaba.html.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  * `access_secret` - (Required) The access secret for the Alibaba account.
+* `organizational_unit_id` - (optional) Organizational unit id.
+
+## Attributes Reference
+
+* `vendor` - The cloud provider ("Alibaba")
+* `alibaba_account_id` - Alibaba account id.
+* `creation_date` - Date the account was onboarded to Dome9
+* `organizational_unit_path` - Organizational unit path
+* `organizational_unit_name` - Organizational unit name
```
Comment (itzikYeret) [inline website/docs/r/cloudaccount_alibaba.html.markdown:44]: I think we have more Attributes Reference

**Review by itzikYeret — CHANGES_REQUESTED** [review]

File: website/docs/r/cloudaccount_alibaba.html.markdown (hunk 1 above)
Comment (chkp-dolevh) [reply website/docs/r/cloudaccount_alibaba.html.markdown]: The brunch change did not keep the file up to date. Done

File: website/docs/r/cloudaccount_alibaba.html.markdown (hunk 2 above)
Comment (chkp-dolevh) [reply website/docs/r/cloudaccount_alibaba.html.markdown:44]: Nope. That correct

File: website/docs/r/cloudaccount_alibaba.html.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+* `name` - (Required) The name of the Alibaba account in Dome9
+* `credentials` - (Required) Has the following arguments:
+    * `access_key` - (Required) The access key for the Alibaba account.
+    * `access_secret` - (Required) The access secret for the Alibaba account.
+* `organizational_unit_id` - (optional) Organizational unit id.
+
+## Attributes Reference
+
+* `vendor` - The cloud provider ("Alibaba")
```
Comment (itzikYeret) [inline website/docs/r/cloudaccount_alibaba.html.markdown:40]: the reference should be exactly like the data source

**Review by itzikYeret — CHANGES_REQUESTED** [review]

**Review by itzikYeret — APPROVED** [review]

**Pull request merged**